doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -533,39 +533,41 @@ The soil moisture deficit :math:`D_{irrig}` is
 
    D_{irrig} = \left\{
    \begin{array}{lr}
-   w_{thresh} - w_{avail} &\qquad w_{thresh} > w_{avail} \\
+   w_{target} - w_{avail} &\qquad w_{thresh} > w_{avail} \\
    0 &\qquad w_{thresh} \le w_{avail}
    \end{array} \right\}
 
-where :math:`w_{thresh}` is the irrigation moisture threshold (mm) and :math:`w_{avail}` is the available moisture (mm). The moisture threshold is
+where :math:`w_{target}` is the irrigation target soil moisture (mm)
 
 .. math::
    :label: 25.62
 
-   w_{thresh} = f_{thresh} \left(w_{target} - w_{wilt}\right) + w_{wilt}
+   w_{target} = \sum_{j=1}^{N_{irr}} \theta_{target} \Delta z_{j} \ .
 
-where :math:`w_{target}` is the irrigation target soil moisture (mm)
+The irrigation moisture threshold (mm) is
 
 .. math::
    :label: 25.63
 
-   w_{target} = \sum_{j=1}^{N_{irr}} \theta_{target} \Delta z_{j} \ ,
+   w_{thresh} = f_{thresh} \left(w_{target} - w_{wilt}\right) + w_{wilt}
 
-:math:`w_{wilt}` is the wilting point soil moisture (mm)
+where :math:`w_{wilt}` is the wilting point soil moisture (mm)
 
 .. math::
    :label: 25.64
 
    w_{wilt} = \sum_{j=1}^{N_{irr}} \theta_{wilt} \Delta z_{j} \ ,
 
 and :math:`f_{thresh}` is a tuning parameter.  The available moisture in
-the soil is
+the soil (mm) is
```

Comment:
- [ ] Looks like I missed the manual line break here. Could you please move this line up to the end of the previous one?

Reply:
Looks like I caused this.  Fixed now.